src/core/components/AMInstallButton/index.js

```diff
@@ -47,6 +47,7 @@ import './styles.scss';
 type Props = {|
   ...WithInstallHelpersInjectedProps,
   addon: AddonType,
+  buttonLabel?: string,
```

Comment:
how about we call it `defaultButtonText` because that's what it is, no?

Reply:
I see what you're saying (since it's under default in the switch) but I guess I don't think of this as default really because it's not used if it's not passed in.

Reply:
mmh hat's the default state, though